tests/forms_tests/tests/test_formsets.py

```diff
@@ -1180,49 +1107,38 @@ def test_html_safe(self):
         self.assertEqual(str(formset), formset.__html__())
 
 
-data = {
-    'choices-TOTAL_FORMS': '1',  # the number of forms rendered
-    'choices-INITIAL_FORMS': '0',  # the number of forms with initial data
-    'choices-MIN_NUM_FORMS': '0',  # min number of forms
-    'choices-MAX_NUM_FORMS': '0',  # max number of forms
-    'choices-0-choice': 'Calexico',
-    'choices-0-votes': '100',
-}
-
-
-class FormsetAsFooTests(SimpleTestCase):
-    def test_as_table(self):
+class FormsetAsTagTests(SimpleTestCase):
+    def test_as_tag(self):
```

Comment:
I'd rather a setUp method and keep the assertions in separate tests since they're not dependent on each other.

Reply:
Agreed :+1: , fixed.